src/org/labkey/test/components/ui/navigation/NavBar.java

```diff
@@ -85,5 +92,8 @@ public WebDriver getDriver()
         public WebElement userIcon = Locator.tagWithAttribute("img", "alt", "User Avatar").findWhenNeeded(this);
         public WebElement projectNameDisplay = Locator.tagWithClass("span", "project-name").findWhenNeeded(this);
         public Input searchBox = Input.Input(Locator.tagWithClass("input", "navbar__search-input"), getDriver()).findWhenNeeded(this);
+        public WebElement searchForm = Locator.tagWithClass("form", "navbar__search-form").findWhenNeeded(this);
+        public WebElement findAndSearchMenuButton = Locator.tagWithId("button", "find-and-search-menu").findWhenNeeded(searchForm);
+        public WebElement findSamplesOption = Locator.linkContainingText("Find Samples").findWhenNeeded(searchForm);
```

Comment:
Just a FYI...
Using findWhenNeeded will wait to find the element on the first time it is referenced, but it only finds the element once. Using refindWhenNeeded will refind the element each time it is referenced. This has obvious perf issues, but if the container is invalidated and refreshed using refind will protect against a staleElement exception.
 
With that said it is unlikely the NavBar will be refreshed without an entire page refresh. So this is fine.